### ISSUE TITLE
dramatiq-gevent support

### ISSUE THREAD

**Issue by FinnLidbetter** [Flared/dramatiq-abort#11]
When starting workers with dramatiq-gevent, this abort middleware will not work. This is because the raise_thread_exception function is not compatible with gevent.
Recently, I submitted PRs to the main dramatiq project to add gevent support to the TimeLimit and ShutdownNotifications middleware (see [https://github.com/Bogdanp/dramatiq/pull/408](https://github.com/Bogdanp/dramatiq/pull/408) and [https://github.com/Bogdanp/dramatiq/pull/426](https://github.com/Bogdanp/dramatiq/pull/426)). These PRs have been accepted and merged.

Would you accept a PR to this project to add support for dramatiq-gevent?

P.S., thanks for creating this middleware extension to dramatiq!

**Comment by isra17** [comment]
Yes, feel free to open a PR with the gevent support. I suppose an approach similar to the manager to abstract away both implementation would work.

Thanks!
